libs/coroutines/include/hpx/coroutines/coroutine.hpp

```diff
@@ -47,6 +47,7 @@
 #include <utility>
 
 namespace hpx { namespace threads { namespace coroutines {
+
```

Comment:
Just so we don't go back and forth on formatting changes like this: I tend to remove empty lines like this :) Happy to leave them in the future if you'd prefer?

Reply:
Uhh. I personally prefer to have them. But I can live with it if we decided to remove them.

Reply:
No worries, let's keep it. I won't be deleting the ones that are there, but I may not always remember to add them either to new code. This is really no big deal.